app/src/main/java/com/tr4n/moviedb/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.tr4n.moviedb
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        supportActionBar?.hide()
+
+        Handler(Looper.getMainLooper()).postDelayed({
+            val mainIntent = Intent(this, HomeActivity::class.java)
+            startActivity(mainIntent)
+            finish()
+        }, 2000)
     }
-}
\ No newline at end of file
+}
```

Comment:
Em tạo Splash Activity riêng 
Main Activity riêng
Splash làm theo hướng dẫn nhé : https://medium.com/android-news/right-way-to-create-splash-screen-on-android-e7f1709ba154